lib/url.c

```diff
@@ -962,21 +963,36 @@ socks_proxy_info_matches(const struct proxy_info *data,
 #define socks_proxy_info_matches(x,y) FALSE
 #endif
 
-/* A connection has to have been idle for a shorter time than 'maxage_conn' to
-   be subject for reuse. The success rate is just too low after this. */
+/* A connection has to have been idle for a shorter time than 'maxage_conn', or
+   created less than 'maxlifetime_conn' ago, to be subject for reuse. The
+   success rate is just too low after this. */
```

Comment:
The success rate comment refers to the idle time part, shouldn't this then make the comment instead say:
~~~c
/* A connection has to have been idle for a shorter time than
  'maxage_conn' (the success rate is just too low after this) or
  created less than 'maxlifetime_conn' ago, to be subject for reuse. */